README.md

```diff
@@ -1,54 +1,186 @@
-
-
-## Built With
-- RUBY
-
-## Project Setup
-To get a local copy up and running follow these simple example steps.
-
-### Setup
-1. Clone the repo in your local machine:
-```bash
-https://github.com/altontonn/Catalog-of-my-things.git
-```
-2. Install the linters
-```bash
-`gem install --no-document rubocop`
-```
-
-## Authors
-
-👤 **Newton Alumasa**
-
-- GitHub: [@github](https://github.com/altontonn/)
-- Twitter: [@twitter](https://twitter.com/AlumasaNewton)
-- Linkedin:[Username](https://www.linkedin.com/in/newton-alumasa/)
-
-👤 **Muskan Gupta**
-
-- GitHub: [@github](https://github.com/muskan2532150)
-- Twitter: [@twitter](https://twitter.com/muskan2532150)
-
-👤 **Moathal Kachi**
-
-- GitHub: [@github](https://github.com/Moathal)
-- Twitter: [@twitter](https://twitter.com/mo_athal)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-- [Ruby capstone](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md)
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Catalog of my Life Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog of my Life Project] <a name="about-project"></a>
+
+**[Catalog of my Life Project]** is a capstone project that uses console app to keep a record of different types of things you own: books, music albums, movies, and games. 
+
+## 🛠 Built With <a name="built-with">Ruby</a>
+
+### Tech Stack <a name="tech-stack">Postgres</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link]()
+
+## Presentation
+[video link](https://drive.google.com/file/d/1ZRHc5NvadcGHRZa6YwhIG0Ycg7XP0RDS/view?usp=share_link)

```

Comment:
- [ ] Nice working on your project and your video presentation, Kindly fix as a requirement the maximum length of your video presentation should be `5 minutes`, yours is `6.56 minutes`.

Reply:
Hello @danielufeli, would you please approve the ask my coding partners to present again later because some of them will not be available?

Reply:
ok re-submit

Reply:
alright @danielufeli, thanks